doc/intro.rst

```diff
@@ -41,23 +41,23 @@ Philosophy
 Installation
 ------------
 
-+-------------------+-----------------------------+-----------------------------------------+
-|Distro             |Helpful Command              |Useful Note                              |
-+===================+=============================+=========================================+
-|**Arch Linux**     |``$ pacaur -S py3status``    |Stable updates. Official releases.       |
-+                   +-----------------------------+-----------------------------------------+
-|                   |``$ pacaur -S py3status-git``|Hot updates. Directly from master branch.|
-+-------------------+-----------------------------+-----------------------------------------+
-|**Debian & Ubuntu**|``$ pypi-install py3status`` |Usually in `python-stdeb` package.       |
-+-------------------+-----------------------------+-----------------------------------------+
-|**Fedora**         |``$ dnf install py3status``  |                                         |
-+-------------------+-----------------------------+-----------------------------------------+
-|**Gentoo Linux**   |``$ emerge -a py3status``    |                                         |
-+-------------------+-----------------------------+-----------------------------------------+
-|**Pypi**           |``$ pip install py3status``  |                                         |
-+-------------------+-----------------------------+-----------------------------------------+
-
-Installation Note: This list is incomplete. Please help us fill in the gaps.
```

Comment:
I think we should encourage people to add more and contribute to the docs, please leave this or rephrase it.

Reply:
I put it there. I think this makes us look a bit desperate... which we aren't.... especially with the `Please help us fill in the gaps` message. If the users want to add something to this list, they can make a pull request... or if they don't... we can look into it ourselves.

Also, we should look into making a `man` page for `py3status`.... and next, make `py3status` available in many distributions (Debian, Ubuntu, ArchLinux, Gentoo, etc)... as I find that they're officially not in there.

Reply:
lol I don't think this looks desperate but alright it's no big deal anyway